CONTRIBUTING.md

```diff
@@ -39,3 +40,59 @@ To reactivate the virtual environment:
 (host)$ pip install tox
 (host)$ tox
 ```
+
+
+## Backporting commits
+
+For the purpose of these instructions, it is assumed a remote named `upstream` points to this repository and a remote named `origin` points at your fork. We want to backport commits from `devel` to the `release_2.0` branch.
+
+```
+origin	git@github.com:me/ansible-runner.git (fetch)
+origin	git@github.com:me/ansible-runner.git (push)
+upstream	https://github.com/ansible/ansible-runner.git (fetch)
+upstream	https://github.com/ansible/ansible-runner.git (push)
+```
+
+### Using Cherry Picker 🍒⛏
```

Comment:
Assuming #778 gets merged and the GH App added, consider adding a section "for maintainers" with just one list item "add a label and follow the bot comments".